packages/echarts/tests/basic.html

```diff
@@ -0,0 +1,186 @@
+<!--
+SPDX-FileCopyrightText: 2023 Siemens AG
+
+SPDX-License-Identifier: MIT
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/echarts@5.4.1/dist/echarts.min.js"></script>
+    <style>
+      body {
+        height: 100vh;
+        width: 100vw;
+        margin: 0;
+      }
+
+      #main {
+        min-height: 100%;
+        min-width: 100%;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="main"></div>
+    <script type="module">
+      import { registerTheme } from '../dist/index.esm.js';
+
+      function getTheme() {
+        const searchParams = new URLSearchParams(location.search);
+
+        if (!searchParams.has('theme')) return 'brand-dark';
```

Comment:
Avoid inline if statements. Hard to read. I will adapt eslint regarding this behaviour. (https://eslint.org/docs/latest/rules/curly)